src/desktop/apps/artist/routes.coffee

```diff
@@ -19,13 +19,13 @@ sd = require('sharify').data
 
 @index = (req, res, next) ->
   tab = if req.params.tab? then req.params.tab else ''
-  includeJSONLD = res.locals.sd.REFLECTION
+  isReqFromReflection = res.locals.sd.REFLECTION
```

Comment:
Could this sort of logic be moved into the split-testing helpers? It strikes me as a very common case that Reflection renders a page and would want to (1) be served the "control" test version and (2) would want to avoid affecting A/B analytics. The same goes for the common bots/crawlers.

Reply:
Yes, that makes sense, if that something we want to do for all tests. Since the split test helper class is used both server and client side. Also I'll check if the reflection flag is available in `sharify` otherwise we'll have to figure out another way to get that value.

Reply:
Great idea @joeyAghion !

I think this should work: https://github.com/artsy/force/blob/master/src/lib/setup.js#L199-L205 mounts the `locals` middleware (which sets some sharify data such as `REFLECTION`), above the `splitTest` one. 

So, within split test this value should already be accessible (and presumably the order of mounting could be changed if needed, but I don't think it is). Maybe the split test setup can add an additional field: we already have `edge` (to direct admins), maybe `reflection`? That would mean send the crawler down this path.

Reply:
Ok sounds good, I moved the reflection check to `splitTest` and added a key called `control_group` which defaults to `control`.

@mzikherman If you can check it out 🙏🏾